frontend/styles/layout.css

```diff
@@ -1,13 +1,13 @@
-$column: 73px;
-$gutter: 47px;
+$column: calc(100%/12);
```

Comment:
That's not how it works. That's why we always ask column and gutter sizes beforehand... I can fix it later.

How you defined it, it will be 90px in 1440px but another width in 1280, and we don't want that.

Reply:
Throw me a solution there doctor, shall we use 90px fixed then?

Reply:
Yes Doctor. Proceed please

Reply:
It's done